apps/dashboard/src/lib/components/Org/PathWay/api.ts

```diff
@@ -0,0 +1,28 @@
+import { supabase } from '$lib/utils/functions/supabase';
+import { get } from 'svelte/store';
+import { isOrgAdmin } from '$lib/utils/store/org';
+
+export async function fetchPathways(profileId: string | undefined, orgId: string | undefined) {
+  if (!orgId || !profileId) return;
+
+  const match: any = {};
+  // Filter by profile_id if role isn't admin within organization
+  if (!get(isOrgAdmin)) {
+    match.member_profile_id = profileId;
+  }
+
+  const { data: allPathways } = await supabase
+    .rpc('get_pathways', {
```

Comment:
Please generate the migration for this function and every other function used in this code and add it in this migration file `20240716161106_pathway.sql`